File: SelectionSort/SelectionSort2.c
#include<stdio.h>

void selectionSort(int *data,int size)
{
int e,f,g,si,oep,iep;
oep=size-2;
iep=size-1;
e=0;
while(e<=oep)
{
f=e+1;
si=e;
while(f<=iep)
{
if(data[f]<data[si]) si=f;
f++;
}
g=data[si];
data[si]=data[e];
data[e]=g;
e++;
}
}

int main()
{
int x[5],y,req;
req=5;
y=0;
while(y<req)
{
printf("Enter a number : ");
scanf("%d",&x[y]);
y++;
}
selectionSort(x,req);
for(y=0;y<req;y++) printf("%d\n",x[y]);
return 0;
}

File: SelectionSort/SelectionSort2.py
def selectionSort(x):
    size=len(x)
    oep=size-2
    iep=size-1
    e=0
    while e<=oep:
        si=e
        f=e+1
        while f<=iep:
            if x[f]<x[si]: si=f
            f+=1
        g=x[e]
        x[e]=x[si]
        x[si]=g
        e+=1
x=[]
req=5
for i in range(req): x.append(int(input("Enter a number : ")))
selectionSort(x)
for i in x: print(i)
